lib/auto-reveal-in-sidebar.js

```diff
@@ -25,15 +25,18 @@ module.exports = {
 
     activate: function() {
         if (!this.observer) {
-            this.observer = atom.workspace.onDidOpen(function(e) {
+            this.observer = atom.workspace.onDidChangeActivePaneItem(function(e) {
                 var workspaceView, activePane;
 
                 if(isTreeViewOpen()) {
                     workspaceView = atom.views.getView(atom.workspace);
                     activePane = atom.workspace.getActivePane();
-                    atom.commands.dispatch(workspaceView, 'tree-view:reveal-active-file');
-                    if (activePane) {
-                        activePane.activate();
+                    // system panes have a uri
+                    if (!activePane.activeItem.uri) {
```

Comment:
this should help avoid unnecessary when when switching to a pane that wouldn't exist in the tree, like system or package screens.